extensions/amp-accordion/1.0/test/test-amp-accordion.js

```diff
@@ -516,6 +517,94 @@ describes.realWin(
       });
     });
 
+    describe('display locking', () => {
+      let defaultCssSupports;
+      let defaultBeforeMatch;
+
+      beforeEach(async () => {
+        toggleExperiment(win, 'amp-accordion-display-locking', true);
+        element = html`
+          <amp-accordion>
+            <section>
+              <h2>Section 1</h2>
+              <div>Puppies are cute.</div>
+            </section>
+            <section expanded>
+              <h2>Section 2</h2>
+              <div>Kittens are furry.</div>
+            </section>
+            <section expanded>
+              <h2>Section 3</h2>
+              <div>Elephants have great memory.</div>
+            </section>
+          </amp-accordion>
+        `;
+        defaultCssSupports = win.CSS.supports;
+        defaultBeforeMatch = win.document.body.onbeforematch;
+      });
+
+      afterEach(() => {
+        win.CSS.supports = defaultCssSupports;
```

Comment:
please also toggleExperiment off in this block

Reply:
Added in the latest.